From 2454e655a8ebac2e5d9aa4561b02b38b2033bc0e Mon Sep 17 00:00:00 2001
From: Kunhui-Li <kunhuix.li@intel.com>
Date: Thu, 19 Aug 2021 15:36:10 +0800
Subject: [PATCH 14/14] config_tools: update kernel type

1. update kern_type to KERNEL_ELF.
2. update kern_mod to Zephyr_ElfImage.

Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>
---
 misc/config_tools/data/adl-s-crb/hybrid.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/misc/config_tools/data/adl-s-crb/hybrid.xml b/misc/config_tools/data/adl-s-crb/hybrid.xml
index ae5e5aeb9..8728d1c32 100644
--- a/misc/config_tools/data/adl-s-crb/hybrid.xml
+++ b/misc/config_tools/data/adl-s-crb/hybrid.xml
@@ -91,8 +91,8 @@
         </memory>
         <os_config>
             <name>Zephyr</name>
-            <kern_type>KERNEL_ZEPHYR</kern_type>
-            <kern_mod>Zephyr_RawImage</kern_mod>
+            <kern_type>KERNEL_ELF</kern_type>
+            <kern_mod>Zephyr_ElfImage</kern_mod>
             <ramdisk_mod/>
             <bootargs></bootargs>
             <kern_load_addr>0x1000</kern_load_addr>
-- 
2.25.1

